src/helpers/response.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable require-jsdoc */
+export default class Response {
```

Comment:
This is nice but is this a team standard? You will want to ensure the rest of the team members will use it

Reply:
We just agreed to implement this way. 
They will update once my PR is merged. Or can they copy?